monika-config-schema.json

```diff
@@ -31,20 +33,27 @@
         "required": ["url"],
         "properties": {
           "method": {
+            "title": "HTTP Method",
             "type": "string",
             "description": "The http method",
-            "enum": ["GET", "POST", "DELETE", "PUT", "PATCH"]
+            "enum": ["GET", "POST", "DELETE", "PUT", "PATCH"],
+            "default": "GET"
           },
           "url": {
+            "title": "Url",
             "type": "string",
-            "description": "The remote URL address to probe"
+            "description": "The remote URL address to probe",
+            "default": "https://api.github.com/zen",
+            "examples": ["https://github.com", "https://httpbin.org/status/200"]
           },
           "timeout": {
+            "title": "Timeout",
             "type": "integer",
-            "description": "Set the timout for the request in seconds",
+            "description": "Set the timeout for the request in seconds",
             "default": 10
           },
           "saveBody": {
+            "title": "SaveBody",
```

Comment:
maybe better: "Save body to db"

Reply:
this is just title actually, the description is below "should reponse body be save in the database"